packages/storage-plus/src/keys.rs

```diff
@@ -250,7 +250,8 @@ impl<T: Endian> From<T> for IntKey<T> {
 
 impl<T: Endian> From<Vec<u8>> for IntKey<T> {
     fn from(wrap: Vec<u8>) -> Self {
-        // TODO: assert proper length
+        // TODO: Consider properly handling case, when `wrap` has length not conforming for the
```

Comment:
Fair enough. Let's do look at TryFrom at some point...

Reply:
Solution here is in fact not to implement `From`, and implement `TryFrom` instead. That's what Rust std does.